### PR TITLE
Rebase dev containers on OBS built containers

### PR DESCRIPTION
**Pull request by coolo**
Don't duplicate containers on dockerhub and obs - but just use the obs containers for both testing and development.

The Rakefile also contains provisioning for deployment, I'm not sure how to handle this.

### CODE REVIEW

**Comment by hennevogel** [comment]
Please let's do this step after we are done with the 2.10 release. We have enough other stuff on our plate right now.

**Comment by vpereira** [comment]
@hennevogel looks like the openSUSE:Leap:42.3 was disabled on `OBS:Server:Unstable`. I tested this PR  locally now and worked flawless. Can we merge it? Otherwise we cannot build our docker containers locally anymore.

**Comment by vpereira** [comment]
i rebased it with master

File: src/api/docker-files/Dockerfile [hunk 1]
```diff
@@ -3,11 +3,19 @@
 # contained rails app generating files in the git checkout with
 # some strange user...
 
-FROM openbuildservice/frontend-base
+FROM registry.opensuse.org/obs/server/unstable/container/leap151/containers/openbuildservice/frontend-base
```
Comment (hennevogel) [inline src/api/docker-files/Dockerfile]: The container on docker hub is "just" a gem cache and rebuilds every time we commit. The container on OBS doesn't.

**Comment by hennevogel** [comment]
@coolo  I would also appreciate if you could stick to smaller pull requests in the future that change one thing....

File: src/api/docker-files/Dockerfile (hunk 1 above)
Comment (hennevogel) [reply src/api/docker-files/Dockerfile]: Ah you do already, sorry

File: src/api/docker-files/Dockerfile [hunk 2]
```diff
@@ -3,11 +3,19 @@
...
 ARG CONTAINER_USERID
 
+# install foreman from rpms, we only need it in docker
+RUN zypper ar http://download.opensuse.org/repositories/devel:/languages:/ruby:/extensions/openSUSE_Leap_15.1/ dlre
+RUN zypper -n --gpg-auto-import-keys ref
+RUN zypper -n in -C 'rubygem(foreman)' 'rubygem(mailcatcher)'
+
+# for lint task
+RUN npm install -g jshint
```
Comment (hennevogel) [inline src/api/docker-files/Dockerfile]: Why do you move this here instead?

Comment (vpereira) [reply src/api/docker-files/Dockerfile]: what would be the alternative  @hennevogel ?

Comment (hennevogel) [reply src/api/docker-files/Dockerfile]: Leave it where it was, in the frontend-base container

Comment (hennevogel) [reply src/api/docker-files/Dockerfile]: All of us rebuild this container all the time.

Comment (coolo) [reply src/api/docker-files/Dockerfile]: that would require jshint to exist as obs package - you can't npm install on obs

Comment (DavidKang) [reply src/api/docker-files/Dockerfile]: @hennevogel, so all the lines 9-15 should be part of frontend-base, right?

Comment (vpereira) [reply src/api/docker-files/Dockerfile]: @DavidKang we will need the following repository `devel:languages:nodejs` to be able to install `jshint`

Comment (hennevogel) [reply src/api/docker-files/Dockerfile]: That package is broken, all of the packages on OBS are...

Comment (hennevogel) [reply src/api/docker-files/Dockerfile]: Guess we have to life with this here now until we find a solution...

**Pull request merged**